ACPs/194-streaming-asynchronous-execution/README.md

```diff
@@ -19,12 +19,21 @@ Validation of transactions to be pushed to the queue is lightweight but guarante
 1. Concurrent consensus and execution streams eliminate node context switching, reducing latency caused by each waiting on the other.
 In particular, "VM time" (akin to CPU time) more closely aligns with wall time since it is no longer eroded by consensus.
 This increases gas per wall-second even without an increase in gas per VM-second.
-2.  Lean, execution-only clients can rapidly execute the queue agreed upon by consensus, providing accelerated receipt issuance and state computation.
+2. Lean, execution-only clients can rapidly execute the queue agreed upon by consensus, providing accelerated receipt issuance and state computation.
 Without the need to compute state _roots_, such clients can eschew expensive Merkle data structures.
 End users see expedited but identical transaction results.
 3. Irregular stop-the-world events like database compaction are amortised over multiple blocks.
-4. EOA-to-EOA transfers of value can be considered settled immediately upon being pushed to the queue by consensus.
-5. Introduces bursty throughput under temporary high load, preemptively accepting transactions that may take marginally longer to execute, without a reduction in security guarantees.
+4. Introduces additional bursty throughput by eagerly accepting transactions, without a reduction in security guarantees.
```

Comment:
"Additional" with respect to what?

```suggestion
4. Introduces bursty throughput by eagerly accepting transactions, without a reduction in security guarantees.
```

Reply:
ACP-176 already supports bursty throughput

Reply:
Which will be replaced by this because $C$ / the leaky bucket won't be used. Not important enough to block this PR, but noting anyway.

Reply:
I think the reason I don't like "Introduces bursty throughput" is because it makes it sound like there wasn't previously bursty throughput.

Like, we could make this say that this design doesn't remove the already existing bursty throughput? But that doesn't really seam like a "Performance improvement".

Reply:
Lets leave it as "additional" for now as it's otherwise too bike-shed-y.